packages/components/src/feature_alert/index.jsx

```diff
@@ -0,0 +1,2 @@
+export { default as Tooltip } from './tooltip';
```

Comment:
why don't you put the component in here?

Reply:
1) Couldn't figure out how to make 2 components work within one story
2) Wasn't thinking about point 1 since thought it'll be ok to move it, since we are moving away `Tooltip` component, and this case looks pretty similar to me

Reply:
ok